android/app/src/main/AndroidManifest.xml

```diff
@@ -24,6 +24,15 @@
         android:name="android.hardware.camera.autofocus"
         android:required="false" />
 
+    <!-- Usage of @react-native-community/netinfo, rn-fetch-blob and Flipper
+         adds the permission requirement for ACCESS_WIFI_STATE in the final 
+         AndroidManifest of the app which makes wifi hardware a compulsary 
+         requirement to install the app. The following statement makes the 
```

Comment:
Let's have this commit message link to the same piece of docs as the previous one did:
https://developer.android.com/guide/topics/manifest/uses-feature-element

Actually, better yet, to the relevant section within that doc:
https://developer.android.com/guide/topics/manifest/uses-feature-element#permissions

It's really pretty counterintuitive, at least to me, that the permission implicitly means a hard requirement for the hardware feature, so seeing the doc helps a lot.